Cargo.toml

```diff
@@ -9,9 +9,10 @@ documentation = "https://docs.rs/dolly"
 keywords = ["gamedev", "camera", "3d"]
 categories = ["game-development"]
 readme = "crates-io.md"
+rust-version = "1.58.1"
```

Comment:
Copied this from `glam`, but I don't recall any of the needed `const` machinery being introduced in that version. Regardless, `dolly` should probably check+enforce some MSRV in its CI (perhaps other code requires higher Rust versions already).